CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 All notable changes to this project will be documented in this file.
+## [2.3.0] - 2020-06-26
+### Added
+- Zoom Liveness flow update
```

Comment:
Quite pedantic, but this doesn't sound like an `Added` - probably should be a `Fixed` or `Changed`?

Reply:
yeah, it is pedantic...but agree ;) Changed